src/util/db/__jest__/IgnoreList.test.ts

```diff
@@ -0,0 +1,17 @@
+import { add, remove, exists } from '../IgnoreList';
+
+jest.mock('../connectionSettings');
```

Comment:
@markokajzer Let me know what you think

Reply:
Hey man 👋 I'd rather not have a file that only has a string in it 😕 I'd like if you could just mock the whole `connection` file, so that a different thing is used for the testing.